tests/ui/abi/invalid-call-abi.rs

```diff
@@ -0,0 +1,14 @@
+// Fix for #142969.
```

Comment:
```suggestion
// Fix for #142969 where an invalid ABI in a signature still had its call ABI computed
// because CTFE tried to evaluate it, despite previous errors during AST-to-HIR lowering.
```